src/libs/antares/InfoCollection/StudyInfoCollector.cpp

```diff
@@ -1,13 +1,15 @@
-#include <numeric>
-#include <antares/benchmarking/info_collectors.h>
-#include <antares/config.h>
-#include <antares/Enum.hpp>
-#include <antares/study/area.h>
+//
+// Created by marechaljas on 22/08/23.
+//
+
+#include "antares/infoCollection/StudyInfoCollector.h"
+#include "antares/benchmarking/DurationCollector.h"
+#include "antares/benchmarking/file_content.h"
+#include "../../../config.h"
```

Comment:
Relative include

Reply:
Yes. config.h is not part of any library so we can't use chevron (<,>) to include it.
We could do so by adding a traget_include_directory instruction but it would open the way to include anything else.